src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -1,30 +1,52 @@
 import React, { useContext } from "react";
 import { s } from "./";
 import Link from "COMPONENTS/Link/";
-import { EpisodesContext as Episodes } from "COMPONENTS/Episodes/";
+import { 
+	EpisodesContext as Episodes 
+} from "COMPONENTS/Episodes/";
+import UTILS from "SHARED/utils.js";
 
 function renderPaginationLink(data, index){
 
 	// CONTEXT
 	// --------------------------
-	const { activeIndex } = useContext(Episodes);
+	const { 
+		get: {
+			activeIndex 
+		},
+		set: {
+			activeIndex: setActiveIndex
+		}
+	} = useContext(Episodes);
 
 	// RENDER
 	// --------------------------
-	const { number } = data;
+	const { 
+		number: episodeNo,
+		title
+	} = data;
+
+	const episodeId = UTILS.convertToSafeString(title, "-");
 
 	const isActive = index === activeIndex;
 
+	// CLICK HANDLER
+	// --------------------------
+	function handlePageClick(){
```

Comment:
change this to a better function name - `handlePageClick` isn't much better than `doTheThing()` ya know?

Reply:
Done.